### DISCUSSION TITLE
Support KLOR

### DISCUSSION THREAD

**Discussion by azzamsa** [manna-harbour/miryoku#342] [Q&A] [answered]
Hi, 👋🏽 

Would you like to add a "proper" KLOR support?

Currently, I get the firmware by merging [this commit](https://github.com/manna-harbour/miryoku_qmk/commit/f8bc600c775f91d5de0d6b2f936cd44511592c51) and [this commit](https://github.com/manna-harbour/miryoku_qmk/commit/32c0d067ae795f212253c9ed3f95bc9dc621d08a). Would you add both of them to `master`?

I have tested both commits, It works fine. But sometimes it registers a key even if I don't press anything. I have no clue if is this caused by the firmware or a hardware error.

Context:

![image](https://github.com/user-attachments/assets/2211efc8-7835-48dd-aba3-dd514bf06b1b)
[source](https://discord.com/channels/939959680611020840/942289799555989515/1147477286640160828)

**Comment by manna-harbour** [comment, marked as answer]
That comment is still the current situation. Once Miryoku QMK is moved to a "userspace" repo (https://github.com/manna-harbour/miryoku/discussions/287) the "[miryoku] Add klor" commit can be added. After that, support for outboards (https://github.com/manna-harbour/miryoku_zmk#outboards) will be added to Miryoku QMK, and then the "[keyboard] Add klor" commit can be replaced with an outboard.

**Reply by azzamsa** [reply]
Thank you!
